### ISSUE TITLE
VPN apps

### ISSUE THREAD

**Issue by ghost** [nextdns/dns-bypass-methods#11]
If I download Mullvad, IVPN, Proton apps to my device it bypasses the "Block bypass methods" isn't that feature supposed to stop the VPN apps from working?

**Comment by Bismuthfallacy** [comment]
> If I download Mullvad, IVPN, Proton apps to my device it bypasses the "Block bypass methods" isn't that feature supposed to stop the VPN apps from working?

Kind of. The ‘block bypass methods’ feature blocks URLs. So it will block the Mullvad website, but the app doesn’t really call the website so the app will probably still work (I say probably because I think it depends on the VPN app and how complex it is). 

If you are using an iPhone and want to prevent bypasses of your NextDNS configuration using VPN apps, you will need to utilise device restrictions to prevent this. I’ve written a guide on how to do this here[https://www.buymeacoffee.com/mindfuliOS/a-set-help-detach-addictive-tendencies-around-iphone-ipad-use#comment-127538](here). 

Basically the NextDNS ‘block bypass methods’ works for proxy websites (though they’re a bit slow on keeping this list updated!). To tackle app functionality, NextDNS can’t really help you entirely and you need to restrict VPN use at the device level (as explained in my guide). 

Hope this helps

**Comment by victoriouspeach28** [comment]
Thank you that is awesome!
